README.md

```diff
@@ -104,6 +106,8 @@ yarn build
 yarn start
 ```
 
+The Swagger UI is available at `http://localhost:3333/api-docs`.
```

Comment:
I noticed that this line is duplicated in the `README.md`. Could you consolidate it into a single, common sentence for the Swagger UI endpoint instead of duplicating it?